### PR TITLE
Extend Sighthound documentation.

### PR DESCRIPTION
**Pull request by kifort**
## Proposed change

Extend the documentation with

- configuration verification
- image processing details with a simple example

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

The change has to be tested and corrected, as I can't test it myself.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @kifort, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17932).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by kifort** [comment]
Hi @robmarkcole, I created this documentation PR as you suggested at. https://github.com/home-assistant/home-assistant.io/issues/17927. If you have the time, please verify and correct it if needed.

File: source/_integrations/sighthound.markdown [hunk 1]
```diff
@@ -63,3 +63,47 @@ source:
       required: false
       type: string
 {% endconfiguration %}
+
+To verify the integration, check if new entities appear as `camera.sighthound` and `image_processing.sighthound_camera_my_cam`
```
Comment (robmarkcole) [inline source/_integrations/sighthound.markdown]: If `camera.sighthound` is created this is new behaviour to me

File: source/_integrations/sighthound.markdown [hunk 2]
```diff
@@ -63,3 +63,47 @@ source:
...
+
+## Process an Image
+
+When you want to process an image, you have to call `image_processing.scan` service and listen to the sighthound.person_detected and/or sighthound.vehicle_detected events.
```
Comment (robmarkcole) [inline source/_integrations/sighthound.markdown]: Calling `scan` is the same for all image processing integrations

File: source/_integrations/sighthound.markdown [hunk 3]
```diff
@@ -63,3 +63,47 @@ source:
...
+
+A simple example is via 2 two automations:
+
+- The first automation is triggered, when a Unify G4 doorbell detects a motion. It calls `image_processing.scan` service to send the camera image to the sighthound server for processing.
```
Comment (robmarkcole) [inline source/_integrations/sighthound.markdown]: Probably dont need to mention the brand of doorbell, just say its motion triggered

**Comment by robmarkcole** [comment]
Im all for adding example automations

File: source/_integrations/sighthound.markdown (hunk 1 above)
Comment (kifort) [reply source/_integrations/sighthound.markdown]: Fixed. It turned out to be a misconfiguration on my side.

File: source/_integrations/sighthound.markdown (hunk 2 above)
Comment (kifort) [reply source/_integrations/sighthound.markdown]: Not everybody is familiar with the image processing integrations. I can remove, if it bothers you.

File: source/_integrations/sighthound.markdown (hunk 3 above)
Comment (kifort) [reply source/_integrations/sighthound.markdown]: Done.

File: source/_integrations/sighthound.markdown [hunk 4]
```diff
@@ -16,7 +16,7 @@ This integration adds an image processing entity where the state of the entity i
 
 If `save_file_folder` is configured, on each new detection of a person, an annotated image with the name `sighthound_{camera_name}_latest.jpg` is saved in the configured folder if it doesn't already exist, and overwritten if it does exist. The saved image shows the bounding box around detected people and can be displayed on the Home Assistant front end using a [Local File](/integrations/local_file/) camera, and used in notifications. If `save_timestamped_file` is configured as `true`, then the annotated image is saved with a file name that includes the time of detection.
 
-**Note** that by default the component will not automatically scan images, but requires you to call the `image_processing.scan` service, e.g.,  using an automation triggered by motion.
+**Note** that by default the component will not automatically scan images, but requires you to call the `image_processing.scan` service, e.g. using an automation triggered by motion.
```
Comment (frenck) [inline source/_integrations/sighthound.markdown]:
```suggestion
**Note** that by default the component will not automatically scan images, but requires you to call the `image_processing.scan` service, e.g., using an automation triggered by motion.
```

File: source/_integrations/sighthound.markdown [hunk 5]
```diff
@@ -63,3 +63,47 @@ source:
...
+
+A simple example is via 2 two automations:
+
+- The first automation is triggered, when a motion is detected. It calls the `image_processing.scan` service to send the camera image to the sighthound server for processing.
+
+- The second automation is triggered by a `sighthound.vehicle_detected` event. It sends a notification to a phone.
+
+```yaml
+# Example automations.yaml entry
+- id: '...'
```
Comment (frenck) [inline source/_integrations/sighthound.markdown]: Please format this according to our documentation and YAML standards.
For more information, check out developer portal

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/sighthound.markdown (hunk 5 above)
Comment (kifort) [reply source/_integrations/sighthound.markdown]: Thank you for the review @frenck. I didn't know about the Home Assistant developer portal. I have read the documentation standards and the YAML Style Guide. I hope that the updated sample confirms with them.

Funnily, my automations.yaml, generated by Home Assistant, uses single quotes for strings.

File: source/_integrations/sighthound.markdown [hunk 6]
```diff
@@ -63,3 +63,45 @@ source:
...
+
+- The first automation is triggered, when a motion is detected. It calls the `image_processing.scan` service to send the camera image to the sighthound server for processing.
+
+- The second automation is triggered by a `sighthound.vehicle_detected` event. It sends a notification to a phone.
+
+```yaml
+# Example automations.yaml entry
+- id: "SOME_UNIQUE_ID"
+  alias: Entrance Motion Image Processing
+  description: "Send a camera image to sighthound, when motion is detected at the entrance."
```
Comment (klaasnicolaas) [inline source/_integrations/sighthound.markdown]:
```suggestion
  description: Send a camera image to sighthound, when motion is detected at the entrance.
```

File: source/_integrations/sighthound.markdown [hunk 7]
```diff
@@ -63,3 +63,45 @@ source:
...
+    domain: binary_sensor
+  action:
+  - service: image_processing.scan
+    target:
+      entity_id: image_processing.sighthound_my_cam
+  mode: single
+
+- id: "ANOTHER_UNIQUE_ID"
+  alias: Arriving Vehicle Notification
+  description: "Send a notifocation to a phone, when a vehicle is detected at the entrance."
```
Comment (klaasnicolaas) [inline source/_integrations/sighthound.markdown]:
```suggestion
  description: Send a notifocation to a phone, when a vehicle is detected at the entrance.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/sighthound.markdown (hunk 2 above)
Comment (frenck) [reply source/_integrations/sighthound.markdown]:
```suggestion
When you want to process an image, you have to call `image_processing.scan` service and listen to the `sighthound.person_detected` and/or `sighthound.vehicle_detected` events.
```

File: source/_integrations/sighthound.markdown [hunk 8]
```diff
@@ -63,3 +63,45 @@ source:
...
       type: string
 {% endconfiguration %}
+
+To verify the integration, check if a new entity is appeared as `image_processing.sighthound_my_cam`
+
+## Process an Image
+
+When you want to process an image, you have to call `image_processing.scan` service and listen to the sighthound.person_detected and/or sighthound.vehicle_detected events.
+
+A simple example is via 2 two automations:
```
Comment (frenck) [inline source/_integrations/sighthound.markdown]: "Simple" is subjective, and can be a bit problematic for users that don't feel like it is simple.

```suggestion
An example using two automations:
```

File: source/_integrations/sighthound.markdown [hunk 9]
```diff
@@ -63,3 +63,45 @@ source:
...
+  - type: motion
+    platform: device
+    device_id: YOUR_DEVICE_ID
+    entity_id: binary_sensor.my_motion_sensor
+    domain: binary_sensor
+  action:
+  - service: image_processing.scan
+    target:
+      entity_id: image_processing.sighthound_my_cam
+  mode: single
```
Comment (frenck) [inline source/_integrations/sighthound.markdown]: YAML styling guidelines

```suggestion
  alias: "Entrance Motion Image Processing"
  description: "Send a camera image to sighthound, when motion is detected at the entrance"
  trigger:
    - type: motion
      platform: device
      device_id: YOUR_DEVICE_ID
      entity_id: binary_sensor.my_motion_sensor
      domain: binary_sensor
  action:
    - service: image_processing.scan
      target:
        entity_id: image_processing.sighthound_my_cam
  mode: single
```

File: source/_integrations/sighthound.markdown [hunk 10]
```diff
@@ -63,3 +63,45 @@ source:
...
+  alias: Arriving Vehicle Notification
+  description: Send a notifocation to a phone, when a vehicle is detected at the entrance.
+  trigger:
+  - platform: event
+    event_type: sighthound.vehicle_detected
+  action:
+  - service: notify.mobile_app_my_iphone
+    data:
+      message: Somebody has just arrived by car.
+  mode: single
```
Comment (frenck) [inline source/_integrations/sighthound.markdown]:
```suggestion
- id: "ANOTHER_UNIQUE_ID"
  alias: "Arriving Vehicle Notification"
  description: "Send a notification to a phone, when a vehicle is detected at the entrance"
  trigger:
    - platform: event
      event_type: sighthound.vehicle_detected
  action:
    - service: notify.mobile_app_my_iphone
      data:
        message: "Somebody has just arrived by car."
  mode: single
```

**Review by frenck — APPROVED** [review]
Thanks, @kifort 👍

**Comment by kifort** [comment]
With pleasure @frenck 
In case you want to "un-blacklist" the Sighthound repo at ~~hassio~~ HACS, please re-assign https://github.com/hacs/default/pull/958 to you. The owner check is currently failing.

**Comment by frenck** [comment]
@kifort that is HACS, which is not related to this project, and not in our control either. Sorry, can't help you there

**Comment by kifort** [comment]
@frenck this HACS PR is in your control:

Error: 'kifort' is not a contributor to 'robmarkcole/HASS-Sighthound'
Information: https://hacs.xyz/docs/publish/include#check-owner

I leave this up-to-you if you want to be back in HACS. I can't push it further.

**Comment by frenck** [comment]
@kifort It is not. This is not the HACS project.

**Pull request merged**